test/e2e/flowaggregator_test.go

```diff
@@ -138,12 +138,19 @@ func TestFlowAggregator(t *testing.T) {
 		}
 	})
 
-	// InterNodeFlows tests the case, where Pods are deployed on different Nodes and their flow information is exported as IPFIX flow records.
+	// InterNodeFlows tests the case, where Pods are deployed on different Nodes
+	// and their flow information is exported as IPFIX flow records.
 	t.Run("InterNodeFlows", func(t *testing.T) {
+		// In worker nodes, Flow Exporter on Antrea Agent is not able to resolve
+		// the DNS name of Flow Aggregator Service. It is flaky depending on the
+		// underlay connectivity in the Kind cluster. Please note that CoreDNS Pods
+		// are deployed on master node because of connectivity issues between kube-apiserver
+		// and worker nodes.
+		skipIfProviderIs(t, "kind", "DNS resolution of Flow Aggregator Service is flaky on worker nodes")
```

Comment:
I notice that you are only skipping inter-node tests (`InterNodeFlows` and `RemoteServiceAccess`). Given that I am more inclined to think that this is another instance of issue https://github.com/vmware-tanzu/antrea/issues/897?

Reply:
This happened with AntreaPolicy enabled case too whose tunnel is vxlan, so not sure if that is the root cause.